iib/workers/config.py

```diff
@@ -285,13 +300,17 @@ def validate_celery_config(conf: app.utils.Settings, **kwargs) -> None:
     if not conf.get('iib_api_url'):
         raise ConfigError('iib_api_url must be set')
 
+    if not conf.get('iib_default_opm'):
```

Comment:
so that means we don't need to ensure its set in playbooks as well?

Reply:
If we set `iib_default_opm` in `Config` class then it is optional and does not have to be set playbook. 
For me it is preferred since it will keep setting of IIB backwards compatible.

Reply:
I do not have strong opinions on this one so I am good either way. But if we start doing validations for one default config value, we are setting a precedent for all. I'll leave this call on Tulsi

Reply:
@chandwanitulsi I guess I did not read it after myself. What I meant is that we do not need validation if the value is present since it will be there by default.

Reply:
Thank you :)